features/FxA-15-OAuth-Management/README.md

```diff
@@ -0,0 +1,83 @@
+Disconnect OAuth services from FxA
+======================
+
+https://mozilla.aha.io/features/FXA-15
```

Comment:
It's probably going to be more useful to link to e.g. https://github.com/mozilla/fxa-features/issues/20 rather than the soon-to-be-defunct Aha card.